integrations/access/accessrequest/app.go

```diff
@@ -488,6 +496,21 @@ func (a *App) updateMessages(ctx context.Context, reqID string, tag pd.Resolutio
 	return nil
 }
 
+func (a *App) getLoginsByRole(ctx context.Context, req types.AccessRequest) (map[string][]string, error) {
+	loginsByRole := make(map[string][]string)
+
+	for _, role := range req.GetRoles() {
+		currentRole, err := a.apiClient.GetRole(ctx, role)
+
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		loginsByRole[role] = currentRole.GetLogins(types.Allow)
```

Comment:
That was not part of the request, but it would make sense to list other logins besides unix logins. I don't think we need to add additional logins into this PR, but definitely open to discussing what the UI should look like here.

I'd be concerned about displaying too much information with all the different types of logins. Maybe we can make this a configuration option to toggle logins for unix, db, windows, etc...
```toml
[logins]
unix = true
windows = true
db = true
...
```

This request was made because the user wanted to be able to quickly identify if any roles granted access to highly privileged logins like `root`. So maybe instead of listing all logins, we could also configure the messages to warn if these types of logins are being requested.
```toml
[logins]
unix = ["root"]
windows = ["superuser"]
...
```

Reply:
I think it's fine as is.
We can always revisit this if needed.

As for your suggestions, I like the latter and we can be backwards compatible by defaulting to `unix = ["*"]`